MediaRecorder.bs

```diff
@@ -510,18 +520,40 @@ interface MediaRecorder : EventTarget {
   <dt><dfn method for="MediaRecorder"><code>isTypeSupported(DOMString type)
     </code></dfn></dt>
   <dd>
+  This method is deprecated; it exists for legacy compatibility reasons only.
+  The <dfn>list of synchronously exposed codecs</dfn> is: <code>vp8</code>,
+  <code>vp9</code>, <code>h264</code>, and <code>opus</code>.
```

Comment:
Let me check if Chrome supports AV1 already with this. If it does, let's add it to this list.
Otherwise, LGTM.

Reply:
Chrome does support av1.
`MediaRecorder.isTypeSupported("video/webm;codecs=av1")` returns `true`